securedrop/requirements/develop-requirements.txt

```diff
@@ -11,50 +11,55 @@ argh==0.26.2              # via sphinx-autobuild, watchdog
 asn1crypto==0.22.0        # via cryptography
 babel==2.4.0              # via sphinx
 backports-abc==0.5        # via tornado
-boto==2.47.0
-certifi==2017.4.17        # via requests, tornado
-cffi==1.10.0              # via cryptography
+bcrypt==3.1.3             # via paramiko
+boto==2.48.0
```

Comment:
could you maybe add one line in the commit comment explaining why the dependencies are updated as well ?